keymaster/ta/asn1.c

```diff
@@ -422,10 +418,6 @@ TEE_Result TA_gen_attest_rsa_cert(const TEE_TASessionHandle sessionSTA,
 		goto error_1;
 	}
 
-	DMSG("mbedTLS certificate: \n");
```

Comment:
Taking into account that a debug verbosity can be configured globally for the whole TA, why not to keep all these `DMSG` ? A think there probably will be cases when we need to check what TA generates.